### PR TITLE
codelist links / extra info

### PR DESCRIPTION
**Pull request by HelenCEBM**
Make codelists sections easier to find/navigate between
Added info on using draft codelists and manually adding codelists

### CODE REVIEW

File: docs/codelist-project.md [hunk 1]
```diff
@@ -16,7 +16,9 @@ opensafely/asplenia/2020-06-02
 opensafely/current-asthma/2020-05-06
 ```
 
-## Adding a CSV to a file
+If necessary, during initial development you can even import codelists this way while they are in draft, but ensure they are finalised and updated in your study before running in the real data. (They may take the form `username/[your_username]/your_codelist/XYZ`)
+
+## Adding/updating a codelist CSV
```
Comment (Jongmassey) [inline docs/codelist-project.md:21]: `## Adding/updating a codelist CSV file` ?

File: docs/codelist-project.md [hunk 2]
```diff
@@ -25,7 +27,11 @@ opensafely codelists update
 
 Beware that in Windows, if one or more of these codelist files is open then this command won't be able to run; close them first.
 
+If necessary, you can also import CSVs not via OpenCodelists - just manually copy the CSV files into `codelists/`. However, we would recommend uploading these to OpenCodelists to import them as above. Note, if you are _also_ using some codelists from OpenCodelists, any manually imported codelists should be stored in a `local_codelists` folder so that they are not overwritten in the next step. 
+
 You will need to add and commit these changes and push to GitHub. 
 If you don't, or a newer version is available than that committed, the automated tests will fail with an error message.
 
+See more on using Codelists in your study definition in [Working with codelists](study-def-codelists).
```
Comment (Jongmassey) [inline docs/codelist-project.md:35]: 404 error when clicking this link on the preview version of the site

File: docs/study-def-codelists.md [hunk 3]
```diff
@@ -7,7 +7,10 @@ To help you create, edit, and manage codelists, OpenSAFELY provides a web-based
 ## Pulling codelists into your study definition
 
 Many functions for defining variables take *codelists* as arguments.
-Codelists live as CSV files in the `codelists/` directory, and are loaded into variables like this:
+
+Codelists live as CSV files in the `codelists/` directory (see instructions in the [Adding codelists to a project](codelists-project) page).
```
Comment (Jongmassey) [inline docs/study-def-codelists.md:11]: 404 error when clicking this link on the preview version of the site

**Review by Jongmassey — CHANGES_REQUESTED** [review]
Couple of links not working when I view this on the preview build of the site

**Comment by HelenCEBM** [comment]
Thanks @Jongmassey hopefully fixed now?

**Comment by Jongmassey** [comment]
[build failed](https://github.com/opensafely/documentation/runs/3768875236)

**Review by Jongmassey — APPROVED** [review]
Links work

**Comment by HelenCEBM** [comment]
huzzah!

**Pull request merged**